api/api-doc/system.json

```diff
@@ -194,6 +194,21 @@
                "parameters":[]
             }
          ]
+      },
```

Comment:
might want to end the single quote in the commit message. so it reads:
```
r'Feature (.*)_SSTABLE_FORMAT is enabled' is the log files.
```

and s/is/in/ in the line above.